src/Microsoft.TestPlatform.CrossPlatEngine/Adapter/FrameworkHandle.cs

```diff
@@ -99,7 +101,20 @@ public int LaunchProcessWithDebuggerAttached(string filePath, string workingDire
     /// <inheritdoc />
     public bool AttachDebuggerToProcess(int pid)
     {
-        return ((ITestRunEventsHandler2)_testRunEventsHandler).AttachDebuggerToProcess(pid);
+#if NETSTANDARD1_3
```

Comment:
Can we encapsulate this in a separate method since the functionality to infer the framework name is not strictly tied to the debugger attaching workflow ? Maybe we can even reuse it later ?

Reply:
Yes, once we have this sorted out with VS.